src/pybind/mgr/dashboard/frontend/src/app/shared/services/api-interceptor.service.ts

```diff
@@ -131,26 +104,8 @@ export class ApiInterceptorService implements HttpInterceptor {
               timeoutId = this.notificationService.notifyTask(finishedTask);
               break;
             case 401:
-              if (this.dashboardClustersMap.size > 1) {
-                this.multiClusterService.setCluster(this.localClusterDetails).subscribe(() => {
-                  localStorage.setItem('cluster_api_url', this.localClusterDetails['url']);
-                });
-                this.multiClusterService.refresh();
-                this.summaryService.refresh();
-                const currentRoute = this.router.url.split('?')[0];
-                if (currentRoute.includes('dashboard')) {
-                  this.router.navigateByUrl('/pool', { skipLocationChange: true }).then(() => {
-                    this.router.navigate([currentRoute]);
-                  });
-                } else {
-                  this.router.navigateByUrl('/', { skipLocationChange: true }).then(() => {
-                    this.router.navigate([currentRoute]);
-                  });
-                }
-              } else {
-                this.authStorageService.remove();
-                this.router.navigate(['/login']);
-              }
```

Comment:
what was the reason this was initially added? and did you verify if that behaviour is working as intended?

Reply:
This was added to avoid the hub cluster's logout if the token of the remote cluster expires. I have removed it for now. I will add a modified fix in this PR - https://github.com/ceph/ceph/pull/56190